layers/+lang/emacs-lisp/packages.el

```diff
@@ -46,6 +47,19 @@
 (defun emacs-lisp/post-init-eldoc ()
   (add-hook 'emacs-lisp-mode-hook 'eldoc-mode))
 
+(defun emacs-lisp/init-auto-compile ()
+  (use-package auto-compile
+    :defer t
+    :init
+    (progn
+      (setq auto-compile-display-buffer nil
+            auto-compile-use-mode-line t
+            auto-compile-mode-line-counter t)
```

Comment:
Mmh, sorry to bug you again :blush:  I'm not sure what the established style in Spacemacs is but doesn't this `setq` expression rather belong into `:config`?